reference/tools/files/basic.rst

```diff
@@ -236,6 +236,25 @@ to capture in the ``conandata.yml`` the scm coordinates (like Git remote url and
 recover it later in the ``source()`` method and have reproducible recipes that can build from sources
 without actually storing the sources in the recipe.
 
+.. currentmodule:: conan.tools.files.conandata
+
+.. autofunction:: update_conandata
+
+
+conan.tools.files.trim_conandata()
```

Comment:
Better add a ``preview`` note? (or experimental?, maybe experimental is too much)

Reply:
Yes, I think the feature itself has proven valuable in CCI as a hook enough for it not to be experimental, but agree on the preview, done 👍